src/_pytest/terminal.py

```diff
@@ -212,6 +212,12 @@ def pytest_addoption(parser: Parser) -> None:
         choices=["yes", "no"],
         help="Whether code should be highlighted (only if --color is also enabled)",
     )
+    group._addoption(
+        "--unicode",
+        action="store_true",
+        default=False,
+        help="Use unicode characters for horizontal rules.",
```

Comment:
In prose Unicode should start with a capital letter:

```suggestion
        help="Use Unicode characters for horizontal rules.",
```

Reply:
So should I update the help message, or remove the option entirely?